tests/models/test_image.py

```diff
@@ -279,6 +288,101 @@ def test_image_coordinates():
+        image4 = None
+        try:
+            mjd = 60000.
+            endmjd = 60000.0007
+            kwargs = { 'mjd': 60000.,
+                       'end_mjd': 60000.0007,
+                       'format': 'fits',
+                       'exp_time': 60.48,
+                       'section_id': 'x',
+                       'project': 'x',
+                       'target': 'x',
+                       'instrument': 'x',
+                       'telescope': 'x',
+                       'filter': 'r',
+                      }
+            # RA numbers are made ugly from cos(dec).
+            # image1: centered on 120, 40, square to the sky
+            image1 = Image( 'one.fits', ra=120, dec=40.,
+                            ra00=119.86945927, ra01=119.86945927, ra10=120.13054073, ra11=120.13054073,
+                            dec00=39.9, dec01=40.1, dec10=39.9, dec11=40.1,
+                            provenance=provenance_base, nofile=True, **kwargs )
+            # image2: centered on 120, 40, at a 45° angle
+            image2 = Image( 'two.fits', ra=120, dec=40.,
+                            ra00=119.81538753, ra01=120, ra11=120.18461247, ra10=120,
+                            dec00=40, dec01=40.14142136, dec11=40, dec10=39.85857864,
+                            provenance=provenance_base, nofile=True, **kwargs )
+            # image3: centered offset by (0.025, 0.025) linear arcsec from 120, 40, square on sky
+            image3 = Image( 'three.fits', ra=120.03264714, dec=40.025,
+                            ra00=119.90210641, ra01=119.90210641, ra10=120.16318787, ra11=120.16318787,
+                            dec00=39.975, dec01=40.125, dec10=39.975, dec11=40.125,
+                            provenance=provenance_base, nofile=True, **kwargs )
+            # imagepoint and imagefar are used to test Image.containing and Image.find_containing,
+            # as Image is the only example of a SpatiallyIndexed thing we have so far.
+            # The corners don't matter for these given how they'll be used.
+            imagepoint = Image( 'point.fits', ra=119.88, dec=39.95, 
+                                ra00=-.001, ra01=0.001, ra10=-0.001, ra11=0.001, dec00=0, dec01=0, dec10=0, dec11=0,
+                                provenance=provenance_base, nofile=True, **kwargs )
+            imagefar = Image( 'far.fits', ra=30, dec=-10, 
+                                ra00=0, ra01=0, ra10=0, ra11=0, dec00=0, dec01=0, dec10=0, dec11=0,
+                                provenance=provenance_base, nofile=True, **kwargs )
+            session.add( image1 )
+            session.add( image2 )
+            session.add( image3 )
+            session.add( imagepoint )
+            session.add( imagefar )
+
+            sought = session.query( Image ).filter( Image.containing( 120, 40 ) ).all()
+            assert ( set( [ pathlib.Path( s.filepath ).name for s in sought ] )
+                     == { 'one.fits', 'two.fits', 'three.fits' } )
```

Comment:
```suggestion
            assert {image1.id, image2.id}.issubset([im.id for im in sought])
```

Reply:
Same comment as in the conesearch PR. You can use sets if you have to test multiple images in "sought".

Reply:
We don't want to use a subset here; it's not just that these images are found, it's also that the other ones are not found.  So, the sets have to be equal.

Reply:
see my comment on the other PR.